documentation/manifest.md

```diff
@@ -68,14 +69,15 @@
     - [ ] Team Explorer view changes to Home page
     - [ ] GitHub header and repo information is shown
 - [ ] **Publishing a local repo**
-  - [ ] File - New - Project - Console Application (or any type of project, doesn't matter much)
-  - [ ] Select "Add to source control" from the dialog and click Ok
-  - [ ] Select "Git" from the Choose Source Control dialog
+- [ ] **File - New - Project - Console Application (or any type of project, doesn't matter much)**
+- [ ] **Set "Create new Git Repository" checkbox to checked**
+- [ ] **Click OK button**
```

Comment:
Ah, I see. They were previously indented and then they no longer are. I tried to keep the setup steps outdented the furthest, but if the steps look clearer with an indent, no objections here to do that!